docs/storage/dataset.md

```diff
@@ -31,12 +31,12 @@ There are four ways to access your datasets:
 
 In the [Apify app](https://my.apify.com), you can view your datasets in the [Storage](https://my.apify.com/storage) section under the [Datasets](https://my.apify.com/storage#/datasets) tab.
 
-Only named datasets are displayed by default. Select the `Include unnamed datasets` checkbox to display all of your datasets.
+Only named datasets are displayed by default. Select the *Include unnamed datasets* checkbox to display all of your datasets.
 
 ![Datasets in app]({{@asset storage/images/datasets-app.png}})
 
-To view or download a dataset in the above mentioned formats, click on its `Dataset ID`. In the detail page, you can update the dataset's name (and, in turn, its [retention period]({{@link storage.md#data-retention}})) and
```

Comment:
https://docs.apify.com/storage/key-value-store?version=develop#apify-sdk
This also shows up with weird asterix rendering, as do a few others here and in Key-val store